package.json

```diff
@@ -42,6 +42,7 @@
     "url": "git://github.com/ampersandjs/ampersand-form-view"
   },
   "scripts": {
+    "validate": "jshint .",
```

Comment:
Agreed that's it's very helpful and nice to have a shorthand for it.

The only problem I have with it is that `precommit-hook` will automatically run `jshint .` (or `npm run lint`) as well as `npm run validate` (if it exists), so in this case it would lint twice. This probably should be fixed in `ampersand-state` as well.